CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ _When adding new entries to the changelog, please include issue/PR numbers where
 
 ## 0.11.4 (UNRELEASED)
 
+- Bugfix: Add check for `.bash_profile` for MacOS. [#643](https://github.com/koordinates/kart/issues/643)
 - Support tab completion in powershell for kart. [#643](https://github.com/koordinates/kart/issues/643)
```

Comment:
I'd actually just remove these two lines; the whole tab completion feature is new in this release so we don't need to list all the bugfixes we've applied to it